### PR TITLE
spell errors

### PR DESCRIPTION
**Pull request by tim-zju**
Signed-off-by: tim-zju <21651152@zju.edu.cn>

### CODE REVIEW

File: contributors/devel/on-call-build-cop.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ and [pulls](https://github.com/kubernetes/kubernetes/pulls): Your job is to be
 the first responder to all new issues and PRs. If you are not equipped to do
 this (which is fine!), it is your job to seek guidance!
 
-  * Support issues should be closed and redirected to Stackoverflow (see example
```
Comment (xulike666) [inline contributors/devel/on-call-build-cop.md:15]: please revert this to “StackOverflow” or "stackoverflow".

File: contributors/devel/automation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ We have added an automated [submit-queue]
 (https://github.com/kubernetes/contrib/blob/master/mungegithub/mungers/submit-queue.go)
 to the
 [github "munger"](https://github.com/kubernetes/contrib/tree/master/mungegithub)
-for kubernetes.
+for Kubernetes.
```
Comment (xulike666) [inline contributors/devel/automation.md]: Could you please transfer all of "kubernetes" to "Kubernetes" in this repository？

**Review by xulike666 — CHANGES_REQUESTED** [review]

File: contributors/devel/automation.md (hunk 2 above)
Comment (tim-zju) [reply contributors/devel/automation.md]: i'll delete these operations which are from "kubernetes" to "Kubernetes"

File: contributors/devel/on-call-build-cop.md (hunk 1 above)
Comment (tim-zju) [reply contributors/devel/on-call-build-cop.md:15]: ok, i transfer it to "StackOverflow"

**Review by xulike666 — CHANGES_REQUESTED** [review]
Please do not change the default markdown format such as "-"、"+" and new line format. Check these unnoticed modifications by text-only editor.

**Review by xulike666 — APPROVED** [review]

**Pull request merged**